Makefile

```diff
@@ -33,7 +33,7 @@ setup-minikube: ## Setup a Kubernetes cluster using Minikube
 	bash platform/00-kubernetes-minikube-setup.sh
 
 .PHONY: setup-frsca
-setup-frsca: setup-certs setup-registry setup-tekton-chains setup-spire setup-vault
+setup-frsca: setup-certs setup-registry install-tekton-pipelines setup-tekton-pipelines install-tekton-chains setup-tekton-chains install-spire setup-spire install-vault setup-vault
```

Comment:
Should kyverno be part of frsca? Seems like we'd want it here instead of adding entries to the quickstart.

Reply:
I agree that it should be added. For some reason we did not have it as part of `setup-frsca` so I did not move it. Maybe it was a remanent of the old work, but I will add it in.